### ISSUE TITLE
[Bug]: Botania Mana Tablets despawn after 5+ minutes when left in the world

### ISSUE THREAD

**Issue by JustJinxed** [FTBTeam/FTB-Modpack-Issues#283]
### Modpack

FTB Ocean Block

### Modpack version

1.7.0

### Log Files

None

### Describe the bug

Dropping a Botania mana tablet in a manapool and waiting about 5 minutes causes the mana tablet to despawn

### Steps to reproduce

Setup a standard manapool with spreaders and mana in it, drop empty mana tablet into pool, wait about 5-7 minutes and the mana tablet will despawn.

### Expected behaviour

Botania mana tablets are not supposed to despawn. 

### Additional information

Removed GC command line options, and they don't seem to have any impact on the behavior. As far as I'm aware, mana tablets do not despawn in other packs like Direwolf20's collect or Enigmatica 6. Is there maybe a entity cleaner mod that's impacting them?

**Comment by Raidobw2** [comment]
Seems to be an issue with WYML despawning those mana tablets.
I'm not sure, @Sunekaer is this intended behavior? Mana items (or Terra Shatterer for example) are not supposed to despawn if inside mana pools, under normal circumstances.

If this closes if it's intended, at least it's documented. There's a workaround, to loop the tablets or items using a hopperhock and an open crate, from Botania.

Here's the WYML config, maybe it acts like a lag clear function:
```
// Set the amount of time it takes for an item to despawn in ticks, This can only be reduced (default 6000)
"ITEM_DESPAWN_TIME": 6000
```

**Comment by enkido** [comment]
Ive got the same problem with applied energistics 2 crystal seeds despawning before fully grown.

**Comment by Raidobw2** [comment]
From 1.0.9 and up, WYML now has:
`Added item despawn denylist, to ensure we do not interfere with specific items`

A newer version is now available as well, 1.1.0, on CurseForge. This behaviour can be configured for mana items as well as ae2 crystal seeds.
